packages/connect/src/core/onCallFirmwareUpdate.ts

```diff
@@ -112,6 +102,7 @@ const waitForReconnectedDevice = async (
                 'we were unable to read device.features on the first interaction after seeing it, retrying...',
             );
             try {
+                registerEvents(reconnectedDevice);
```

Comment:
Just asking: is this `registerEvents` here needed even in current state because of `DEVICE.BUTTON` event? Or is it prepared for THP? Anyway, LGTM.